### PR TITLE
Add rename feature to fire-atlas

### PR DESCRIPTION
**Pull request by luanpotter**
fire-atlas-rename-feature

### CODE REVIEW

File: fire_atlas_editor/lib/screens/editor_screen/widgets/rename_atlas_modal.dart [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  }
+
+  Future<void> _renameAtlas(BuildContext ctx) async {
+    final store = SlicesProvider.of<FireAtlasState>(ctx);
+    final newName = newNameController.text;
+    if (newName.isEmpty) {
+      store.dispatch(
+        CreateMessageAction(
+          type: MessageType.ERROR,
+          message: 'You must inform the name to rename.',
```
Comment (spydon) [inline fire_atlas_editor/lib/screens/editor_screen/widgets/rename_atlas_modal.dart]: What does this mean, maybe it can be reformulated slightly? 😄

**Review by spydon — APPROVED** [review]

File: fire_atlas_editor/lib/screens/editor_screen/widgets/rename_atlas_modal.dart (hunk 1 above)
Comment (luanpotter) [reply fire_atlas_editor/lib/screens/editor_screen/widgets/rename_atlas_modal.dart]: it means you must provide a name in order to rename the project. how would you prefer it to be worded? happy to change :)

Comment (luanpotter) [reply fire_atlas_editor/lib/screens/editor_screen/widgets/rename_atlas_modal.dart]: how about `Name is required` ?

Comment (erickzanardo) [reply fire_atlas_editor/lib/screens/editor_screen/widgets/rename_atlas_modal.dart]:
```suggestion
          message: 'The new name is required.',
```

**Review by erickzanardo — APPROVED** [review]
LGTM! A very needed feature!

**Pull request merged**